app/views/deployments.html

```diff
@@ -96,8 +98,66 @@
                   </tr>
                 </tbody>
               </table>
-              <div ng-if="(unfilteredReplicationControllers | hashSize) > 0">
-                <h3>Other replication controllers</h3>
+              <div ng-if="k8sDeployments | hashSize">
+                <h3>Deployments</h3>
+                <table class="table table-bordered table-hover table-mobile">
+                  <thead>
+                    <tr>
+                      <th>Name</th>
+                      <th>Last Version</th>
+                      <th>Replicas</th>
+                      <th>Created</th>
+                      <th>Strategy</th>
+                    </tr>
+                  </thead>
+                  <tbody ng-repeat="k8sDeployment in k8sDeployments | orderObjectsByDate : true">
+                    <tr>
+                      <td data-title="Name">
+                        <a ng-href="{{k8sDeployment | navigateResourceURL}}">{{k8sDeployment.metadata.name}}</a>
+                      </td>
+                      <td data-title="Last Version">
+                        #{{k8sDeployment | annotation : 'deployment.kubernetes.io/revision'}}
+                      </td>
+                      <td data-title="Replicas">
+                        <span ng-if="k8sDeployment.status.replicas !== k8sDeployment.spec.replicas">{{k8sDeployment.status.replicas}}/</span>{{k8sDeployment.spec.replicas}} replica<span ng-if="k8sDeployment.spec.replicas != 1">s</span>
+                      </td>
+                      <td data-title="Created">
+                        <relative-timestamp timestamp="k8sDeployment.metadata.creationTimestamp"></relative-timestamp>
+                      </td>
+                      <td data-title="Strategy">
+                        {{k8sDeployment.spec.strategy.type | sentenceCase}}
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </div>
+              <div ng-if="replicaSets | hashSize" id="replica-sets">
+                <h3>Replica Sets</h3>
```

Comment:
once we do the roll-up under Deployments this should probably say Other Replica Sets, like we do for RCs